### PR TITLE
VACMS-10165: Add lovell documentation.

### LINKED ISSUE #10165: Create Lovell readme in CMS repo
As a developer or anyone associated with the special handling of Lovell I would like a page of documentation that can be updated as special handling is added for Lovell

## ACs 
- [x] Lovell specific page created and linked as a readme within the READMES.
- [x] lovell module help contains a link to this page
- Explains
   - [x] Bifurcation links to FE code where page and menu bifurcation takes place.
   - [x] Menu 
   - [ ] breadcrumbs
   - [x] pathing and aliases

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

Closes #10165 

## QA steps
- [x] Jump links from module help to readme works.
- [x] Jumplinks to page headings work 
- [ ] Validate that content of the documentation  accurately reflects the current-ish state of the FE integration branch
- [ ] Validate that content of the documentation  accurately reflects the current-ish state of the CMS

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: READMES/facilities-lovell.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+
+## Pages
+### CMS
+In the CMS, pages are either Clones or singles.
+   - Clone: A CMS nodes that appears in both subsystems only differing by their path. A clone is any page with field_administration of "Lovell Federal health care" (tid: 347)
+   - Single: A page that only appears in one or the other.  Any page with a field_administration of "Lovell - TRICARE" (tid: 1039) OR "Lovell - VA" (tid: 1040)
+   - Twins: Separate CMS nodes that have a non-identical counterpart in both subsystems.  In the CMS a twin has no connection to its twin.  It has no knowledge that the twin exists.  So a twin is just a single, that has another similar single out there.  Twin is a human construct, not a CMS construct.
+
+### Content-Build
```
Comment (wullaski) [inline READMES/facilities-lovell.md:27]: Order would be:
1. Deep copy any node with a field administration of a clone, to become the TRICARE clones.
2. Alter all paths of clones to reflect Lovell - TRICARE and Lovell - VA.
3. Alter all titles of fieldOffice and fieldRegionPage for use in querying the menus
5. Alterations to titles of some pages to match VA or TRICARE.
6. Put the new TRICARE copies back in the pages data.

File: READMES/facilities-lovell.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+The two different systems are run as separate website sections with some overlap.
+
+To be clear, everything related to Lovell is tech debt because it does not follow any existing patterns.  Every effort has been made to minimize the amount and location of the tech debt, but there is plenty of it both CMS and Content-Build.
+
+
+## Pages
+### CMS
+In the CMS, pages are either Clones or singles.
+   - Clone: A CMS nodes that appears in both subsystems only differing by their path. A clone is any page with field_administration of "Lovell Federal health care" (tid: 347)
```
Comment (wullaski) [inline READMES/facilities-lovell.md:23]: Change nodes to node

File: READMES/facilities-lovell.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+   1. Alterations to titles of some pages to match VA or TRICARE.
+
+## Menus
+
+### CMS
+The menu lovell_federal_health_care is being used as the central menu for both VA and TRICARE. This was done to reduce the maintenance of two separate menus and trying to match the weight and shared items between them.
+An additional field 'field_menu_section' has been added to the lovell_federal_health_care menu.  On node save, the field is set programmatically to either 'both', 'VA', or 'TRICARE' based on what was set on the node for field_administration.
+### Content-Build
+The frontend approach to the menu involves the following:
+
```
Comment (wullaski) [inline READMES/facilities-lovell.md:42]: 2. Deep copy to make VA and TRICARE menus.
...
5. Delete original menu.

**Review by JayDarnell — APPROVED** [review]
Approved now that John's recommendations have been addressed.

**Comment by swirtSJW** [comment]
image

**Pull request merged**